### PR TITLE
Deploy using binary package

### PR DESCRIPTION
**Pull request by yuzhichang**
### What problem does this PR solve?

Add corresponding issue link with summary if exists -->

Issue link:

### Code changes

- [ ] Has Code change
- [ ] Has CI related scripts change

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Note for reviewer

### CODE REVIEW

**Review by writinwaters — CHANGES_REQUESTED** [review]
You can download the binary package (deb, rpm, or tgz) for your respective host operating system from https://github.com/infiniflow/infinity/releases. These packages are designed to be compatible with Linux x86_64 hosts that are no older than CentOS 7. They are statically linked, except for glibc. For example:

File: README.md [hunk 1]
```diff
@@ -44,12 +44,22 @@ Supports a wide range of data types including strings, numerics, vectors, and mo
 
 ## 🎮 Get Started
 
-### Docker pull
+### Deploy Infinity database
```
Comment (writinwaters) [inline README.md:47]: Add a blank line

**Review by writinwaters — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yuzhichang) [reply README.md:47]: Done.

**Comment by yuzhichang** [comment]
@mergify queue

**Pull request merged**